From ccd9feddf71e5bbd841c6f70555bd8317964b9fd Mon Sep 17 00:00:00 2001
From: aswin7469 <aswinas@pixysos.com>
Date: Wed, 16 Aug 2023 22:32:44 +0530
Subject: [PATCH] PixelPropsUtils: Do not spoof ASI

* Non-pixel devices do not need to spoof asi to enable most features
* This fixes now playing in pixel devices since asi is checked with
  device codename to load sountrigger models for now playing

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 core/java/com/android/internal/util/lineage/PixelPropsUtils.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/java/com/android/internal/util/lineage/PixelPropsUtils.java b/core/java/com/android/internal/util/lineage/PixelPropsUtils.java
index ea4ce0a5f62b..9ac69acae12b 100644
--- a/core/java/com/android/internal/util/lineage/PixelPropsUtils.java
+++ b/core/java/com/android/internal/util/lineage/PixelPropsUtils.java
@@ -92,6 +92,7 @@ public class PixelPropsUtils {
             "com.google.android.apps.cameralite",
             "com.google.android.dialer",
             "com.google.android.euicc",
+            "com.google.android.as",
             "com.google.ar.core",
             "com.google.android.youtube",
             "com.google.android.apps.youtube.kids",
-- 
2.25.1

